README.md

```diff
@@ -14,29 +18,64 @@ This project framework provides the following features:
 
 ### Prerequisites
 
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
+* Visual Studio Code
+* Dev Containers support in Visual Studio Code
+* Azure subscription
+* Azure account with Owner permissions on the Subscription
+* Docker runtime
 
 ### Installation
 
-(ideally very short)
+We recommend you fork this project in GitHub and then clone your fork in order to setup GitOps linking to your own fork. This will enable you to apply changes and experiment more easily.
+
+By using the Dev Container, a K3D cluster and any documented client tools will be automatically installed and ready to use for testing out the sample.
+
+* Fork this repository in GitHub.
+* Clone your fork locally.
+* Open the project with Visual Studio Code.
+* Launch the Dev Container with Command Palette > `Dev Containers: Reopen in Container`.
+* Log into Azure and set your default subscription.
+
+  ```bash
+  az login --tenant <your tenant ID or domain>
+
+  az account set -s <your subscription ID or name>`
+  ```
 
-- npm install [package name]
-- mvn install
-- ...
+* Prepare the following inputs to create environment variables in the next step:
+  * `ARC_CUSTOMLOCATION_OID`: retrieve the unique Custom location Object ID for your tenant by running `az ad sp show --id bc313c14-388c-4e7d-a58e-70017303ee3b --query id -o tsv`
+  * `TENANT_ID`: your Microsoft Entra ID Tenant ID
+  * `LOCATION`: make sure you choose something from the supported list of regions as documented in [Deploy Azure IoT Operations, see Location table](https://learn.microsoft.com/en-us/azure/iot-operations/get-started/quickstart-deploy?tabs=codespaces#connect-a-kubernetes-cluster-to-azure-arc)
+  * Use your preferred Azure resource names where you can find the `...` in the variable contents
+
+* Create a new file to load environment variables under `./temp/envvars.sh`, with the following contents:
+
+  ```bash
+  # change the below set to match your environment
+  export RESOURCE_GROUP=rg-...
+  export CLUSTER_NAME=arck-...
+  export AKV_NAME=kv-...
+  export TENANT_ID="<>"
+  export LOCATION=northeurope # your choice
+  export ARC_CUSTOMLOCATION_OID="<>"
+  export DEFAULT_NAMESPACE=azure-iot-operations # do not change
+  ```
+
+* Load the environment variables in your terminal by running:
+
+  ```bash
+  source ./temp/envvars.sh
+  ```
 
 ### Quickstart
-(Add steps to get up and running quickly)
 
-1. git clone [repository clone url]
-2. cd [repository name]
-3. ...
+To enable the local K3D cluster to Azure Arc and install Azure IoT Operations with the default components and configuration.
 
+1. Open a `bash` terminal and leave it in the default root folder `/workspaces/azure-edge-extensions-aio-observability-gitops`.
+1. Run the script `./deploy/1-arc-k8s-connect.sh`. This will take a few minutes.
+1. Tun the script `./deploy/2-azure-iot-operations.sh`. Grab a coffee, this can take 15 minutes.
 
```

Comment:
Could you document somewhere how to view that messages are flowing correctly? So a user knows that everything is running correctly. I can see it in the mqttui like this:
![image](https://github.com/Azure-Samples/azure-edge-extensions-aio-observability-gitops/assets/25376553/82b94198-341a-4199-bd8a-7c38bbad6c53)
Is that correct?

Also I don't see an opc tcp pod
![image](https://github.com/Azure-Samples/azure-edge-extensions-aio-observability-gitops/assets/25376553/ca102651-72e3-4466-b183-cd18d3452e21)
Is that expected?

Reply:
The intention is indeed to document a few extra steps how to validate the installation worked correctly. But I was intending to do this once the base devcontainer was ready so you are not blocked by me. 

The point on OPC TCP missing: that is not expected so I will review and test again. I missed something in the ARM template most probably.